src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4312,10 +4344,16 @@ .FORWARDHELPCATEGORY Cmdlet
             $consoleWidth = [System.Math]::Max([System.Console]::WindowWidth, 20)
 
             if ($pagerArgs) {
-                # Supply pager arguments to an application without any PowerShell parsing of the arguments.
+                # Start the pager arguments directly if the PSNativeCommandArgumentPassing feature is enabled.
+                # Otherwise, supply pager arguments to an application without any PowerShell parsing of the arguments.
                 # Leave environment variable to help user debug arguments supplied in $env:PAGER.
-                $env:__PSPAGER_ARGS = $pagerArgs
-                $help | Out-String -Stream -Width ($consoleWidth - 1) | & $pagerCommand --% %__PSPAGER_ARGS%
+                if ($EnabledExperimentalFeatures -contains 'PSNativeCommandArgumentPassing') {
+                    $help | Out-String -Stream -Width ($consoleWidth - 1) | & $pagerCommand $pagerArgs
+                }
+                else {
+                    $env:__PSPAGER_ARGS = $pagerArgs
+                    $help | Out-String -Stream -Width ($consoleWidth - 1) | & $pagerCommand --% %__PSPAGER_ARGS%
```

Comment:
> & $pagerCommand --% %__PSPAGER_ARGS%

It looks we will get here only if on Unix. `%__PSPAGER_ARGS%` is expanded by PowerShell, right?

Reply:
it should reach this code when the pager is a standalone executable (regardless of platform) PowerShell expands this (with `Environnment.ExpandEnvironmentVariable` before passing it to the native app.